JavaScript/JavaScriptSDK/appInsights.ts

```diff
@@ -312,6 +312,32 @@ module Microsoft.ApplicationInsights {
         }
 
         /**
+         * Sets the autheticated user id and the account id in this session.
+         * User auth id and account id should be of type string. They should not contain commas, semi-colons, equal signs, spaces, or vertical-bars.
+         *   
+         * @param authenticatedUserId {string} - The authenticated user id. A unique and persistent string that represents each authenticated user in the service.
+         * @param accountId {string} - An optional string to represent the account associated with the authenticated user.
+         */
+        public setAuthenticatedUserContext(authenticatedUserId: string, accountId?: string) {
```

Comment:
@olegsych to me personally that's more confusing than setXXX. In knockout () means observable property, I don't think there's anything similar in our api yet.
I like the idea in general though and maybe that's the way we need to go for all context-level properties. Do you think we need a separate issue/user story?